### ISSUE TITLE
'wheel pack' a previously unpacked wheel file raises UserWarning: Duplicate name

### ISSUE THREAD

**Issue by mmunozm** [pypa/wheel#293]
When repacking a previously unpacked wheel, the command
```bash
wheel pack unpacked_wheels/<package>-<version> -d final_wheel_dir/
```
it turns out that zipfile.py raises (on stderr)
```
UserWarning: DuplicateName: '<package>-<version>.dist-info/RECORD'
```

Platform: Windows 10
Python Version: 3.7.2
Wheel Version: 0.33.3

**Comment by agronholm** [comment]
Please provide instructions to reproduce.

**Comment by agronholm** [comment]
So this only happens on Windows. Fixed in 6c8effd79d2c71b768ac606c6b28579433ec62fd.
